packages/shared/lib/core/network/actions/getAndUpdateNodeInfo.ts

```diff
@@ -0,0 +1,15 @@
+import { profileManager } from '@lib/wallet'
+import { get } from 'svelte/store'
+import { INodeInfoResponse } from '../interfaces'
+import { updateNodeInfo } from '../stores/node-info.store'
+
+export async function getAndUpdateNodeInfo(): Promise<void> {
+    let nodeInfoResponse: INodeInfoResponse | null
+    try {
+        nodeInfoResponse = <INodeInfoResponse>(<unknown>await get(profileManager).getNodeInfo())
```

Comment:
I guess we'll have to properly adjust the return type of `getNodeInfo()` here. Maybe put a TODO here, so we don't forget?

Reply:
Done